src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,7 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User user = UserService.findByEmail(email);
```

Comment:
create `userService` as `private final UserService userService` class-level field
```suggestion
        User user = userService.findByEmail(email);
```